unraid_mcp/subscriptions/resources.py

```
@@ -104,22 +104,45 @@ async def logs_stream_resource() -> str:
             }
         )
 
-    def _make_resource_fn(action: str, query: str):
+    def _make_resource_fn(action: str):
```

Ruff (0.15.5)

[warning] Missing return type annotation for private function `_make_resource_fn`

(ANN202)